src/theme/DefaultTheme.js

```diff
@@ -0,0 +1,149 @@
+const core = {
+  white: '#fff',
+  gray: '#565a5c',
+  grayLight: '#82888a',
+  grayLighter: '#cacccd',
+  grayLightest: '#f2f2f2',
+
+  borderMedium: '#c4c4c4',
+  border: '#dbdbdb',
+  borderLight: '#e4e7e7',
+  borderLighter: '#eceeee',
+
+  primary: '#00a699',
+  primaryShade_1: '#33dacd',
+  primaryShade_2: '#66e2da',
+  primaryShade_3: '#80e8e0',
+  primaryShade_4: '#b2f1ec',
+  primary_dark: '#008489',
+
+  secondary: '#007a87',
+
+  yellow: '#ffe8bc',
+  yellow_dark: '#ffce71',
+};
+
+export default {
```

Comment:
One of the things I was thinking about was that... if we're importing this into a project that uses many `react-with-styles` packages and we need to register the theme, we should probably namespace the theme so that there are no collisions. Something like:
```
export default {
  reactDates: { 
    color: ...,
    zIndex: ...,
    ...
  },
};
```
@gabergg @lencioni @ljharb what do you think?

Reply:
Yep, I think it should definitely be namespaced.

Reply:
I'm confused, why should it be namespaced?

This module exports a theme directly; wouldn't whoever registered the theme be able to install it under a key themselves?

Reply:
But the components themselves would be expecting the `color`, `zIndex`, etc. keys directly and there'd be no way to change that.

Reply:
gotcha. maybe it'd be better to have `withStyles` support a "namespace" option or something, but this is fine for now i guess.